starter/src/main/java/com/starter/fullstack/dao/InventoryDAO.java

```diff
@@ -52,7 +52,8 @@ public List<Inventory> findAll() {
    */
   public Inventory create(Inventory inventory) {
     // TODO
-    return null;
+    inventory.setId(null);
+    return this.mongoTemplate.insert(inventory);
```

Comment:
Although this method is named create, the description says that it should be able to update inventory as well. How can we achieve both create and update with as little database actions as possible?